File: ts/src/special-cards/chip-cache.ts
import { SpecialCardDefinition } from './types';

export const chipCacheCard: SpecialCardDefinition = {
  id: 'chip-cache',
  name: 'Chip Cache',
  shortName: 'Cache',
  description: 'Acquire: gain 8 chips immediately.',
  rarity: 'common',
  cost: 6,
  accent: '#f6c453',
  applyOnAcquire: () => ({
    chipsDelta: 8
  })
};

File: godot/Scripts/SpecialCards/ChipCache.cs
namespace Roshambo.SpecialCards;

public static class ChipCache
{
	public static readonly SpecialCardDefinition Definition = new()
	{
		Id = "special:chip-cache",
		Name = "Chip Cache",
		ShortName = "Cache",
		Description = "Gain +8 chips immediately.",
		Rarity = SpecialCardRarity.Common,
		Cost = 6,
		Accent = "#ffd700",
		ApplyOnAcquire = _ => new SpecialCardAcquireResult { ChipsDelta = 8 }
	};
}

